### ISSUE TITLE
Implementing design changes to Standardized Dashboards

### ISSUE THREAD

**Issue by chloedotbrown** [department-of-veterans-affairs/va.gov-team#33939]
## Issue Description
In addition to the need for more trainings and resources, the user research conducted in Sprints 65 & 66 identified design changes that would improve the standardized dashboards and make them more useful to VFS teams.

---
## Resources
- [MURAL board synthesis](https://app.mural.co/t/adhoccorporateworkspace2583/m/adhoccorporateworkspace2583/1637099802029/6e3346bb7da074e665cd38e5f6091ab34f2eaebb?wid=0-1637641937003&outline=open)
- [Research Readout presentation](https://docs.google.com/presentation/d/1nxFQyQmuph6TjnPygNes-Ca23M9Hjcu9rmftAs0UoAM/edit?usp=sharing)
- [Design change wishlist](https://docs.google.com/spreadsheets/d/1RlDgC7dUiuVdfC1Y8kLDx7adqoPkl5qIjSFs-pUsofs/edit?usp=sharing) _(please fill out)_

## Tasks
- [x] Compile [list](https://docs.google.com/spreadsheets/d/1RlDgC7dUiuVdfC1Y8kLDx7adqoPkl5qIjSFs-pUsofs/edit?usp=sharing) of prioritized design changes
- [x] Meet to synthesize lists and identify any blockers
- [x] Assign feasible tasks among Data Analysts and Data Designer
- [x] Implement changes
- [x] QA new cards
- [x] Update Object IDs in [AooA spreadsheet](https://docs.google.com/spreadsheets/d/19cS2nrii95YcMmGcsPQj0RSeSu3Xg6t2ri6KLfJkahw/edit#gid=712826366) to ensure tracking is updated correctly
- [x] Flag any wishlist items for future iterations in the Domo tracking list: **Avg. time to complete forms (corrected)**

## Acceptance Criteria
- [x] Design changes identified in user research and feasible in Domo are implemented to improve user experience of standardized dashboards.
- [x] Design changes identified in user research that are outside the scope of Domo's capabilities are documented.

**Comment by chloedotbrown** [comment]
I've updated the [Design Wishlist spreadsheet](https://docs.google.com/spreadsheets/d/1RlDgC7dUiuVdfC1Y8kLDx7adqoPkl5qIjSFs-pUsofs/edit#gid=1114005375) based on today's meeting with Jonathan, Pavan, and me. Documentation of discussed tasks and prioritization (including tasks that will be out of scope for Q4 work) live in the "Brainstorm" tab, as reference

More importantly, I've also created a new sheet called "Task List" for us to divvy up tasks moving forward. The tasks we okayed for work this sprint are grouped into categories, with signup spots for Task Owners and Reviewers. I've put myself down for the two categories where past context seems like it would be the most useful, but @jonathan-epstein13 and @pavanhothi – please assign yourselves for the things you'd like to be working on, and tag each other or me as your reviewers. 

@michelle-dooley – if there's anything not included in this list you'd like to see added to this year's work, let me know!

**Comment by chloedotbrown** [comment]
All tasks on the [standardized dashboard redesign spreadsheet](https://docs.google.com/spreadsheets/d/1RlDgC7dUiuVdfC1Y8kLDx7adqoPkl5qIjSFs-pUsofs/edit#gid=1114005375) are complete and have been reviewed, except for the following: 

| Status  | Dashboard | Task | Reason  | 
| -------- |  -------- | -------- | -------- |
| ❌ Blocked | Forms | Redo "Avg. Time to Complete" cards | BQ connector and Domo ETL still in development |
| 🌀  Needs data | All | New drillpath charts for device cards  | Waiting on backfill for `device_subcategory` that combines across datasources |
| 🗑️  Removed | All | Add office hours | Top-right resources too crowded, Slackbot may solve  |
| 🗑️  Removed | Search | Add bounce rate | Redundant to "Search abandonment" metrics, bounce not in data  |
| 🗑️  Removed | All | Add trends over time cards | Redundant to existing drillpaths |

However, neither of the first two seem like they should count as _actual_ "blockers" for this redesign. Avg. time to complete is being addressed in #31789 – and in the meantime, we have replaced those cards with "under construction" notecards to avoid anyone using bad data that they shouldn't have access to. Meanwhile, the `device_subcategory` field seems like a "nice to have," but will only appear in card detail views, not on the dashboards themselves. I'll touch base with Julie to see what Jonathan requested and where things were left off.

**Comment by jennaouellet** [comment]
@chloedotbrown QA is complete and all new cards have passed!


## QA

### Forms

Chart title | Assets | Pass / Fail
-- | -- | --
Browser Type Used|  [Custom segment ](https://analytics.google.com/analytics/web/template?uid=qY9ioaAyRiuViY-dwyfIwA)   |  :green_circle: PASS
Devices Used | [Custom segment](https://analytics.google.com/analytics/web/template?uid=qY9ioaAyRiuViY-dwyfIwA) |  :green_circle: PASS

### Search

Chart title | Assets | Pass / Fail
-- | -- | --
Browser Type Used|  [Custom segment](https://analytics.google.com/analytics/web/template?uid=J7NjqYpbRf-6ZaRvVmi8mQ)   |  :green_circle: PASS
Devices Used | [Custom segment](https://analytics.google.com/analytics/web/template?uid=J7NjqYpbRf-6ZaRvVmi8mQ) |  :green_circle: PASS

### Content

Chart title | Assets | Pass / Fail
-- | -- | --
Browser Type Used|  [Custom segment](https://analytics.google.com/analytics/web/template?uid=urKHf17rSCC0q6emrFSiRw)   |  :green_circle: PASS
Devices Used | [Custom segment](https://analytics.google.com/analytics/web/template?uid=XO3JuNQfQm2DXzEqKLg0rQ) |  :green_circle: PASS

**Comment by chloedotbrown** [comment]
Awesome, thanks so much for the speedy QA, @jennaouellet!

@michelle-dooley I've added a couple more items to the [BI Tool wishlist](https://docs.google.com/spreadsheets/d/1qhY7-__GgsDiNfUs5u3z5BTxLfbmdLX2kWuviHMay4A/edit#gid=0), which closes out the last of our Acceptant Criteria. Closing this ticket!
